### DISCUSSION TITLE
ch1-basic/ch1-02-hello-revolution

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#616] [General]
# ch1-basic/ch1-02-hello-revolution

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/ch1-basic/ch1-02-hello-revolution.html

**Comment by codingknees** [comment]
不光是hello world，C的标准库功能也是少的可怜，直到Go。所以涅槃重生的不光是hello world

**Comment by x43125** [comment]
14寸的笔记本，Chrome浏览器打开的时候，像这种标题比较多的右侧跳转到下一页的箭头会被挡住

**Comment by slg33856023** [comment]
go语言 大有可为！！！

**Comment by QiuTF** [comment]
学习
